com.jetbrains.PhpStorm.json

```diff
@@ -153,17 +149,23 @@
             ]
         },
         {
-            "name": "wrapper",
+            "name": "ide-flatpak-wrapper",
             "buildsystem": "meson",
             "config-opts": [
                 "-Deditor_binary=/app/extra/phpstorm/bin/phpstorm",
-                "-Dprogram_name=phpstorm",
-                "-Deditor_title=PhpStorm"
+                "-Dprogram_name=phpstorm-wrapper",
```

Comment:
Name it just `phpstorm`.

Reply:
i think phpstorm-wrapper is better, to distinguish from phpstorm binary executable

Reply:
The phpstorm binary executable don't live in the same directory anyway.

Reply:
yeah, just to be more clear, if you want i can remove "-wrapper"